app/inpututils.h

```diff
@@ -5,6 +5,8 @@
 #include "qgsquickfeaturelayerpair.h"
 #include "qgsquickmapsettings.h"
 
+extern QString LOG_FILE;
```

Comment:
better to define inside InputUtils as a static member rather than global variable